src/ray/object_manager/plasma/store.cc

```diff
@@ -210,6 +210,7 @@ uint8_t *PlasmaStore::AllocateMemory(size_t size, MEMFD_TYPE *fd, int64_t *map_s
 
   // Try to evict objects until there is enough space.
   uint8_t *pointer = nullptr;
+  int num_tries = 0;
```

Comment:
Do we need to retry here? We also retry at a higher level, at the CreateRequestQueue.